backend/apps/llm/views.py

```
@@ -493,6 +495,107 @@ def _test_email(self, channel, test_id, timestamp):
+    def _test_custom_webhook(self, channel, test_id, timestamp):
+        """测试自定义Webhook通知渠道"""
+        try:
+            import requests
+
+            config_dict = channel.config_dict
+            url = config_dict.get('url') or config_dict.get('webhook_url') or config_dict.get('webhook')
+
+            if not url:
+                return Response({
+                    'success': False,
+                    'message': '自定义Webhook URL未配置，请在config_data中设置 url 字段'
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)